src/main/java/com/softserve/teamproject/config/WebSecurityConfig.java

```diff
@@ -1,19 +1,58 @@
 package com.softserve.teamproject.config;
 
+import com.softserve.teamproject.service.impl.UserDetailsServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration
-        .EnableWebSecurity;
-import org.springframework.security.config.annotation.web.configuration
-        .WebSecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.AuthenticationException;
 
+/**
+ * The class provides basic security configuration.
+ */
 @Configuration
 @EnableWebSecurity
-public class WebSecurityConfig  extends WebSecurityConfigurerAdapter {
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        http
-                .authorizeRequests()
-                .antMatchers("/", "/index").permitAll();
-    }
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
```

Comment:
You have different number of blank lines between methods, variables and so on. Somewtimes it's two, sometimes one, in SpringSecurityInit.java you have no blank lines after imports. Could you please make one line in all files (if it's ok with yours code convention)?